utils/collect-updates/lib/collect-dependents.js

```diff
@@ -11,8 +11,10 @@ function collectDependents(nodes) {
       return;
     }
 
-    // depth-first search
+    // breadth-first search
```

Comment:
Thanks for figuring this out! As I've mentioned before, I'm extremely bad at graph theory and related algorithms.